src/components/ShortTextInput.jsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState} from 'react'
+import { Link } from 'gatsby'
+import styled from "styled-components"
+
+const StyledShortTextInput = styled.section`
+    width: 37.5rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    padding: 1em 1em 2em 1em;
+
+  div {
+    width: 23rem;
+    /* height: 15rem; */
+    border: 1px solid var(--dark-grey);
+    border-radius: 2.2px;
+    padding: .2rem 1rem;
+    /* margin: 1rem 1rem 2rem 1rem; */
+  }
+  div:active {
+    border: 2px solid var(--button-outline-blue);
+  }
+
+  div:focus-within {
+    border: 2px solid var(--button-outline-blue);
+  }
+
+  label {
+    font-weight: 300;
+    font-size: .8em;
+    color: var(--dark-grey);
+    display: block;
+    margin-bottom: .1rem;
+  }
+  
+  input {
+    margin-left: .1rem;
+    border: none;
+    resize: none;
+    font-weight: 400;
+    color: var(--tints-blizzard-black);
+  }
+  
+  input:focus {
+    outline: none;
+  }
+  
+  span {
+    width: 23rem;
+    line-height: 1.3em;
+    font-size: .8em;
+    font-weight: 100;
+    color: var(--dark-grey);
+    margin-left: .2rem;
```

Comment:
When determining rem you always want to divide whatever the pixel count is by 16 so if the margin is 4px, you'll want to divide 4/16 for .25rem. It's really not that big of a deal and I doubt any user would be able to really see the difference, but it keeps all the numbers even. I always round to the nearest four decimal places (assuming they are whole pixel numbers that's as small as they should go).

Reply:
got it